components/single_image_view.jsx

```diff
@@ -168,8 +162,7 @@ export default class SingleImageView extends React.PureComponent {
             );
 
             fadeInClass = 'image-fade-in';
-            imageLoadedDimension = {cursor: 'pointer'};
-            imageContainerDimension = {};
```

Comment:
I intentionally clear this style after the image is loaded so that CSS will handle image responsiveness on resize of browser or like expanding/shrinking of RHS.

Reply:
If we want that we need to be 100% sure the CSS height matches with the height we set previously, which it wasn't

Reply:
I did some check and indeed you're correct.